src/Sulu/Component/Symfony/CompilerPass/OverwriteClassCompilerPass.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Symfony\CompilerPass;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+class OverwriteClassCompilerPass implements CompilerPassInterface
```

Comment:
I'm really not a big fan of this compilerpass and if only needed for the test case I would it move there!